ansible/roles/operator-pipeline/templates/openshift/tasks/commit-pinned-digest.yml

```diff
@@ -3,21 +3,99 @@ apiVersion: tekton.dev/v1beta1
 kind: Task
 metadata:
   name: commit-pinned-digest
+  labels:
+    app.kubernetes.io/version: "0.4"
+  annotations:
+    tekton.dev/pipelines.minVersion: "0.21.0"
+    tekton.dev/categories: Git
+    tekton.dev/tags: git
+    tekton.dev/displayName: "commit-pinned-digest"
+
 spec:
   workspaces:
     - name: source
+    - name: ssh-directory
+      optional: true
+      description: |
+        A .ssh directory with private key, known_hosts, config, etc. Copied to
+        the user's home before git commands are executed. Used to authenticate
+        with the git remote when performing the clone. Binding a Secret to this
+        Workspace is strongly recommended over other volume types.
   params:
     - name: dirty_flag
+    - name: GIT_USER_NAME
+      type: string
+      description: |
+        Git user name for performing git operation.
+      default: ""
+    - name: GIT_USER_EMAIL
+      type: string
+      description: |
+        Git user email for performing git operation.
+      default: ""
+    - name: USER_HOME
+      description: |
+        Absolute path to the user's home directory. Set this explicitly if you are running the image as a non-root user or have overridden
+        the gitInitImage param with an image containing custom user configuration.
+      type: string
+      default: "/tekton/home"
+    - name: VERBOSE
+      description: Log the commands that are executed during `git-clone`'s operation.
+      type: string
+      default: "true"
+  results:
+    - name: commit
+      description: The precise commit SHA that was fetched by this Task.
   steps:
     - name: commit-changes
       image: registry.access.redhat.com/ubi8-minimal
```

Comment:
This image doesn't contain the "git" binary. Firstly, test the pipeline with the script and submit the PR when it will successfully create a PR to some repository. Thank you :slightly_smiling_face: